scripts/variantstore/wdl/ImportGenomes.wdl

```diff
@@ -162,20 +172,27 @@ task GetMaxTableId {
 task CreateImportTsvs {
   input {
     File input_vcf
+    File input_vcf_index
```

Comment:
is the vcf index currently a column in the data model? thinking about running this in Terra and whether we can just do `this.samples.vcf_index` or whatever

Reply:
it is a column in the data model